src/status_im/common/feature_unavailable/view.cljs

```diff
@@ -0,0 +1,28 @@
+(ns status-im.common.feature-unavailable.view
```

Comment:
@mohsen-ghafouri I think a better place for this entire `status_im/common/feature_unavailable/` directory would be inside `status_im/contexts/keycard/` because it's not about any generic non-implemented feature, it's only about keycard-related features.